astropy/table/serialize.py

```diff
@@ -165,8 +188,9 @@ def _represent_mixin_as_column(col, name, new_cols, mixin_cols,
         obj_attrs['__info__'] = info
 
     # Store the fully qualified class name
-    obj_attrs.setdefault('__class__',
-                         col.__module__ + '.' + col.__class__.__name__)
+    if not isinstance(col, SerializedColumn):
+        obj_attrs.setdefault('__class__',
+                             col.__module__ + '.' + col.__class__.__name__)
```

Comment:
I'm not sure if this is actually the fully qualified name. The ``col.__module__`` is good, but ``col.__class__.__name__`` misses if the class is defined in aa local context. OTOH, I'm not sure if those can be unserialized...

Reply:
Indeed, they cannot...